.gitignore

```diff
@@ -3,3 +3,4 @@
 /.vscode
 /.vagrant
 .phpunit.result.cache
+composer.lock
```

Comment:
This is intentionally not included in the `.gitignore`, as it is copied when creating a new Laravel Zero project. 👍🏻